README.md

```diff
@@ -14,31 +14,37 @@ Curt Kim, Solomon Montagno, Jose Estrada
 git clone https://github.com/dhk2431/Group_5_Movies_Review_App
 ```
 
-### Install the dependencies:
+### For development in challenges:
 
-```
-cd Group_5_Movies_Review_App
+cd movies-review-app
 yarn install
-```
-
-## Usage
+yarn run dev
 
-### To run the server:
+### Instructions for other developers:
 
-```
-yarn dev
-```
+Clone the project files from github using
+git clone https://github.com/dhk2431/Group_5_Movies_Review_App.git
 
-### To make the database:
+Once user designates file location in terminal,
+cd server
 
-```
+Create a new database called “movies-review-app_development”
 createdb movies-review-app_development
-createdb movies-review-app_test
-createdb movies-review-app_e2e
 
-yarn migrate:make movies
-yarn migrate:latest
-```
+Make sure to yarn install before migrating anything into the database so all the dependencies and packages are configured.
+yarn install
+
+Create a migration for the movies table so that it can take in a required title, year, genre and an optional synopsis and link for a poster of the movie.
+yarn run migrate:make "name of migration for movies table"
+
+Make sure to migrate the table to the database
+yarn run migrate:latest && yarn run migrate:rollback && yarn run migrate:latest
+
+Once all the migrations are set, ensure that the seeded data is added to your database
+yarn db:seed
+
+Lastly, connect to have the server up and running!
+yarn run dev
```

Comment:
lets trim these up

- let's shorten the install context. You can assume that the engineer looking at this project has some basic context already. e.g. you can remove line 34 entirely
- hmm, they shouldnt need to make a migration, you will have done that already. Remove lines 37 & 38 entirely
- testing a rollback is something that the developer does, but doesnt need to be done for anyone else. They will assume that you have made the migration rollback-able. just have them do one simple migration
- replace "Once all the migrations are set, ensure that the seeded data is added to your database" with "Seed the database". They will already assume these work in sequential order
- edit "Lastly, connect to have the server up and running!" to "Start up server"

all yarn commands should be in backticks for proper markdown e.g. `yarn run dev`